geode-core/src/integrationTest/resources/org/apache/geode/codeAnalysis/sanctionedDataSerializables.txt

```diff
@@ -578,9 +502,6 @@ org/apache/geode/internal/admin/remote/LicenseInfoResponse,2
 fromData,18
 toData,15
 
-org/apache/geode/internal/admin/remote/MissingPersistentIDsRequest,1
```

Comment:
this removal of MissingPersistentIDsRequest surprised me.  The class still exists in my checkout and is a DSFID.

Reply:
I re-ran `./gradlew :geode-core:integrationTest --tests=AnalyzeCoreSerializablesJUnitTest` and `sanctionedDataSerializables.txt` did not change. `MissingPersistentIDsRequest` is still not listed there. I see it ultimately derives from `DSFID`.

Reply:
I see why - that class has no fromData/toData methods.  The test just searches for those in parsed class files.  It doesn't care if they're DataSerializable/DSFID.

Reply:
I wonder if it used to have fromData/toData and now does not? why else would it have showed up in this file previously?